staff/albert-fuente/array-functions/array-is-array.js

```diff
@@ -0,0 +1,9 @@
+function isArray(array){
+  if(array===undefined)throw TypeError("ERROR variable no definida" )
```

Comment:
ERROR this is NOT the original behaviour of Array.isArray function, but this: `Array.isArray() // => false`. update your code behaviour accordingly, fully compliance with the native one.